packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info.tsx

```diff
@@ -87,10 +90,18 @@ const Info = observer(({ handleSelect, item, list }: TInfo) => {
                         {is_description_tab_selected ? (
                             <React.Fragment>
                                 <TradeCategoriesGIF category={type.value} selected_contract_type={item?.value} />
-                                <TradeCategories category={type.value} onClick={onClickGlossary} />
+                                <TradeCategories
+                                    category={type.value}
+                                    onClick={onClickGlossary}
+                                    is_multiplier_fx={cached_mult_cancellation_list?.length === 0}
```

Comment:
nit-pick) just a shorter version:
```suggestion
                                    is_multiplier_fx={!cached_mult_cancellation_list?.length}
```